lib/src/platform_client.dart

```diff
@@ -1330,15 +1331,7 @@ class PlatformMessagingKeys {
 }
 
 /// This extension is a way for us to expose and share location update timestamp functionality internally only.
-extension LocationThrottling on PlatformClient {
-  DateTime get lastLocationUpdateTimestamp => _lastLocationUpdateTimestamp;
-  bool get shouldThrottlePositionUpdate {
-    DateTime now = DateTime.now();
-    if (now.difference(_lastLocationUpdateTimestamp).inMilliseconds < 1500) {
-      // Same throttling delay as `KurrentoRoom.updateLocation`.
-      return true;
-    }
-    _lastLocationUpdateTimestamp = now;
-    return false;
-  }
```

Comment:
Since the error was suggesting that we might have an issue with our throttling algorithm, I moved that code in a class for easier re-use and testability.